src/components/App/App.component.jsx

```diff
@@ -1,51 +1,39 @@
-import React, { useState } from 'react';
+import React from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
 
 import AuthProvider from '../../providers/Auth';
 import HomePage from '../../pages/Home';
-import LoginPage from '../../pages/Login';
-// import NotFound from '../../pages/NotFound';
-// import SecretPage from '../../pages/Secret';
-// import Private from '../Private';
-// import Fortune from '../Fortune';
+import Favorites from '../../pages/Favorites';
+import WatchLater from '../../pages/WatchLater';
 import Layout from '../Layout';
 import VideoPlayer from '../../pages/VideoPlayer';
-// import { random } from '../../utils/fns';
+import Private from '../Private';
+import NotFound from '../NotFound';
 
 function App() {
-  const [ term, setTerm ] = useState('wizeline');
-  // useLayoutEffect(() => {
-  //   const { body } = document;
-
-  //   function rotateBackground() {
-  //     const xPercent = random(100);
-  //     const yPercent = random(100);
-  //     body.style.setProperty('--bg-position', `${xPercent}% ${yPercent}%`);
-  //   }
-
-  //   const intervalId = setInterval(rotateBackground, 3000);
-  //   body.addEventListener('click', rotateBackground);
-
-  //   return () => {
-  //     clearInterval(intervalId);
-  //     body.removeEventListener('click', rotateBackground);
-  //   };
-  // }, []);
 
   return (
     <BrowserRouter>
       <AuthProvider>
-        <Layout setTerm={setTerm}>
+        <Layout>
           <Switch>
             <Route exact path="/home">
-              <HomePage term={term}/>
+              <HomePage />
             </Route>
-            <Route exact path="/login">
-              <LoginPage />
-            </Route>
-            <Route exact path="/video">
+            <Route path="/video/:videoId">
               <VideoPlayer />
             </Route>
+            <Route exact path="/notFound">
+              <NotFound />
+            </Route>
```

Comment:
this route should be the last one defined, the switch will render the first component that matches the route, in this case the if the route provided is`/asdf` the switch won't render anything, in this case we need to have a route defined at the end to render the `NotFound`, something like:

```jsx
<Route component={NotFound} />
```